app/code/community/Bolt/Boltpay/Helper/GraphQL/ClientTrait.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * Bolt magento plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2020 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+use Bolt_Boltpay_Helper_GraphQL_Constants as GraphQLConstants;
+
+
+/**
+ * Trait Bolt_Boltpay_Helper_GraphQL_ClientTrait
+ *
+ * Provides utility methods related to the Bolt GraphQL API:
+ *
+ */
+trait Bolt_Boltpay_Helper_GraphQL_ClientTrait
```

Comment:
This is a simple trait.  There is no need for a separate directory for this.  Let's follow convention and place this in the top level directory and call it `Bolt_Boltpay_Helper_GraphQLClientTrait` or `Bolt_Boltpay_Helper_GraphQLTrait`

Reply:
done